### ISSUE TITLE
Suggestion: Scam score based on a screenshots proof, approved by the arbitrators

### ISSUE THREAD

**Issue by slrslr** [haveno-dex/haveno#1457]
**Idea on a reputation (scam score) system using real proof/screenshots approved by arbitrators:**

After a few months from completing reversible payment method trade, money are charge back by a scammer. Victim makes a screenshot proof of the chargeback and also of the initial incoming transaction. Victim finds a scamming trade party in a Haveno trading history and submits a feedback/review along with that screenshots to the community arbitrators.

**Arbitration process:**
If arbitrator dismiss the submission, one more arbitrator needs to confirm the decision (a mechanism against biased arbitrator where can be seen identity of the previous arbitrator?). The screenshots of the scam are presented in a non downloadable and temporary form for privacy. Arbitrators can easily use magnifier to verify screenshots quality and can ask for a new/better screenshot maybe.
**Protection of the feedback arbitration system from misuse:**
1. submitter can only post a feedback on someone if there is a completed/closed trade with that person and if he never submit the feedback request for that person before
2. feedback is dismissed on the poster and arbitrator side(because poster can adjust software) in case the submitter caused two dismissed feedback in last 30 days.
3. arbitration system does not accept/dismiss the feedback request if there is lets say 2 feedback requests open for same submitter.

**Displaying scam score/feedback:**
Approved feedback is visible for anyone under scammer profile, for example "Scammer: 1 case (last: 2024-12-06. 18 fiat trades since then.)". Visitor can click number of cases and see the feedback of the victims. If indefinitely storing feedback is not ideal, maybe showing the number of approved scam cases is enough.

**Block trade offers of a scammers:**
Maybe Haveno should deny submitting new fiat trade offers of the users with 3 scam cases or more and also if scammer modify the software to submit it anyway (if that is possible), other users should not see offers of such scammer. This should be done only if the feedback arbitration system is bulletproof and false feedback can not go through/approved.

**Comment by slrslr** [comment]
@privacyOG  Quote exactly part you are replying to, so I know the context.

**Comment by privacyOG** [comment]
> @privacyOG  Quote exactly part you are replying to, so I know the context.

Im refering to the whole suggestion, a scammer could simply delete and reinstall a new Haveno instance, therefore the scammer will have clean slate. There is no system that 100% perfect

**Comment by slrslr** [comment]
@privacyOG this post was useless for me. You have not quoted my text. What I have described is working on aged accounts, offcourse users can then see if the account is new and decide not to trade with it.

**Comment by privacyOG** [comment]
> @privacyOG this post was useless for me. You have not quoted my text. What I have described is working on aged accounts, offcourse users can then see if the offer maker account is new and decide not to trade with it.

Your lucky im patient
In your first post, first few lines you have written this "After a few months from completing reversible payment method trade, money are charge back by a scammer. Victim makes a screenshot proof of the chargeback and also of the initial incoming transaction".  This means the scammer is the taker ( buyer ) not the maker ( Seller ). 

Based on this I asked again what stops a scammer who will have multiple instances of haveno running at same time delete and reinstall. You can not see the age of the account of someone accepting your offer.

**Comment by slrslr** [comment]
> You can not see the age of the account of someone accepting your offer.

I am not placing offers, instead I am only accepting offers in case i need to accept a reversible payment method, so per my understanding mine mentioned proposal would work and help me better select among old users, because i would not accidentally select old user with bad scam score (as describe din initial post).

Even if i would be maker, then I should be able to deny new (old bad score) users to take my offers.

**Comment by privacyOG** [comment]
> I am not placing offers, instead I am only accepting offers in case i need to accept a reversible payment method, so per my understanding mine mentioned proposal would work and help me better select among old users, because i would not accidentally select old user with bad scam score (as described in initial post).
> 
> Even if i would be maker, then I (my Haveno client app) should be able to deny new (old bad score) users to take my offers.

Appreciate the reply I still cant see it. But i will leave it to other to comment

Also not sure how much of impact it will have on Haveno software where it will remain usable especially over tor where there is this added data and images. 

We really should avoid payment methods that have a history of and or have a high risk of payment reversal.

I will now leave this for others with more knowledge than me to comment on it.

**Comment by boldsuck** [comment]
> Whats stopping the scammer of opening a new instance of Haveno? Its a is easy process to delete reinstall and new instance.

Yeah, he **just needs to delete the hidden service address**, get a new one and can no longer be identified.
Bisq wiki recommends generating a new hidden service address every now and then for privacy reasons.

**Comment by U65535F** [comment]
So it's not possible to stop scammers? I think there are other ways but I am not able to recall them (maybe I'm dumb).

**Comment by boldsuck** [comment]
- **Sellers can't scam, XMR is multisig. Buyers are safe.**
- Buyers can commit fraud in payments with chargeback risk.
Use the recommended precautions or use payment methods without chargeback risk for large amounts.

In over 8 years of Bisq1 and 8 months of RetoSwap, I have never heard of stolen bank accounts being used.
